manuscript/02_webpack_compared.md

```diff
@@ -65,7 +65,7 @@ With Make, you model your tasks using Make-specific syntax and terminal commands
 
 ![Grunt](images/grunt.png)
 
-[Grunt](http://gruntjs.com/) went mainstream before Gulp. Especially its plugin architecture contributed towards its popularity. Plugins are often complex by themselves. As a result when configuration grows, it can become difficult to understand what's going on.
+[Grunt](http://gruntjs.com/) was the first popular task runner for front-end developers. Especially its plugin architecture contributed towards its popularity. Plugins are often complex by themselves. As a result when configuration grows, it can become difficult to understand what's going on.
```

Comment:
Since you’re discussing Gulp later, there is not much point in comparing Grunt to Gulp.